### PR TITLE
feat(route): add 东北大学医学与生物信息工程学院

### PR DESCRIPTION
**Pull request by TennousuAthena**
## 该 PR 相关 Issue / Involved issue

Close #

## 完整路由地址 / Example for the proposed route(s)

```routes
/neu/bmie/news
/neu/bmie/academic
/neu/bmie/talent_development 
/neu/bmie/international_exchange 
/neu/bmie/announcement 
/neu/bmie/undergraduate_dev
/neu/bmie/postgraduate_dev
/neu/bmie/undergraduate_recruit
/neu/bmie/postgraduate_recruit
/neu/bmie/CPC_build
/neu/bmie/CPC_work
/neu/bmie/union_work
/neu/bmie/CYL_work
/neu/bmie/security_management
/neu/bmie/alumni_style
```

## 新 RSS 检查列表 / New RSS Script Checklist
  
- [x] 新的路由 New Route
  - [x] 跟随 [v2 路由规范](https://docs.rsshub.app/joinus/script-standard.html) Follows [v2 Script Standard](https://docs.rsshub.app/en/joinus/script-standard.html)
- [x] 文档说明 Documentation
  - [x] 中文文档 CN
  - [ ] 英文文档 EN
- [x] 全文获取 fulltext
  - [x] 使用缓存 Use Cache
- [ ] 反爬/频率限制 anti-bot or rate limit?
  - [ ] 如果有, 是否有对应的措施? If yes, do your code reflect this sign?
- [x] [日期和时间](https://docs.rsshub.app/joinus/pub-date.html) [date and time](https://docs.rsshub.app/en/joinus/pub-date.html)
  - [x] 可以解析 Parsed
  - [ ] 时区调整 Correct TimeZone
- [ ] 添加了新的包 New package added
- [ ] `Puppeteer`

### CODE REVIEW

File: lib/routes/universities/neu/bmie.js [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+module.exports = async (ctx) => {
+    let type = ctx.params.type;
+    if (typeof map[type] !== 'undefined') {
+        type = map[type];
+    }
+    const newsUrl = `${baseUrl}/${type}/list.htm`;
+    const response = await got({
+        method: 'get',
+        url: newsUrl,
+    });
```
Comment (TonyRL) [inline lib/routes/universities/neu/bmie.js:33]:
```suggestion
    const response = await got(newsUrl);
```

File: lib/routes/universities/neu/bmie.js [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+
+    const title = $('title').text();
+    const items = $('#subIndex > div.main_frame_sub > div.detail_sub > div > div > div > ul > li').slice(0, 7).get();
+    const results = await Promise.all(
+        items.map(async (item) => {
+            const $ = cheerio.load(item);
+            const title = $('a').attr('title');
+            const url = baseUrl + $('a').attr('href');
+            const data = await ctx.cache.tryGet(url, async () => {
+                const result = await got.get(url);
```
Comment (TonyRL) [inline lib/routes/universities/neu/bmie.js:46]:
```suggestion
                const result = await got(url);
```

File: lib/routes/universities/neu/bmie.js [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+                    .find('p')
+                    .each(function () {
+                        $(this).removeAttr('style');
+                        $(this).removeAttr('class');
+                    });
+                $('.wp_art_adjoin').remove();
+                return { description: $('.entry').html(), date, author: auhor };
+            });
+            const description = data.description;
+            const pubDate = data.date;
```
Comment (TonyRL) [inline lib/routes/universities/neu/bmie.js:79]:
```suggestion
            const pubDate = parseDate(data.date);
```
docs: https://docs.rsshub.app/joinus/pub-date.html

File: lib/routes/universities/neu/bmie.js [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+            const author = data.author;
+            const result = {
+                title,
+                description,
+                link: url,
+                pubDate,
+                author,
+            };
+            return Promise.resolve(result);
```
Comment (TonyRL) [inline lib/routes/universities/neu/bmie.js:88]:
```suggestion
            return result;
```

File: docs/university.md [hunk 5]
```diff
@@ -901,6 +901,16 @@ xskb1 对应 <http://www.auto.uestc.edu.cn/index/xskb1.htm>
 
 </Route>
 
+### 东北大学医学与生物信息工程学院
+
+<Route author="TennousuAthena" example="/neu/bmie/news" path="/neu/bmie/:type" :paramsDesc="['种类名']">
+
+| 学院新闻 | 学术科研 | 人才培养 | 国际交流 | 通知公告 | 本科生培养 | 研究生培养 | 本科生招募 | 研究生招募 | 党的建设 | 党委工作 | 工会工作 |   共青团工作 | 安全管理 | 校友风采 |
```
Comment (TonyRL) [inline docs/university.md:908]: Please use a vertical table

**Review by TonyRL — COMMENTED** [review]
Please follow [v2 standard](https://docs.rsshub.app/joinus/script-standard.html).

**Comment by TennousuAthena** [comment]
All checked!
Please review again @TonyRL

File: lib/v2/neu/bmie.js [hunk 6]
```diff
@@ -0,0 +1,93 @@
...
+                    .find('p')
+                    .each(function () {
+                        $(this).removeAttr('style');
+                        $(this).removeAttr('class');
+                    });
+                $('.wp_art_adjoin').remove();
+                return { description: $('.entry').html(), date, author: auhor };
+            });
+            const description = data.description;
+            const pubDate = Date.parse(data.date);
```
Comment (TonyRL) [inline lib/v2/neu/bmie.js:76]:
```suggestion
            const pubDate = parseDate(data.date);
```
docs: https://docs.rsshub.app/joinus/pub-date.html

File: docs/university.md [hunk 7]
```diff
@@ -901,6 +901,30 @@ xskb1 对应 <http://www.auto.uestc.edu.cn/index/xskb1.htm>
 
 </Route>
 
+### 东北大学医学与生物信息工程学院
+
+<Route author="TennousuAthena" example="/neu/bmie/news" path="/neu/bmie/:type" :paramsDesc="['分类id 见下表']">
```
Comment (TonyRL) [inline docs/university.md:906]:
```suggestion
<Route author="TennousuAthena" example="/neu/bmie/news" path="/neu/bmie/:type" :paramsDesc="['分类 id 见下表']">
```

File: lib/v2/neu/radar.js [hunk 8]
```diff
@@ -0,0 +1,13 @@
+module.exports = {
+    'www.bmie.neu.edu.cn': {
+        _name: '东北大学医学与生物信息工程学院',
```
Comment (TonyRL) [inline lib/v2/neu/radar.js:3]:
```suggestion
        _name: '东北大学',
```
`_name` should match the H2 header of the docs

File: lib/v2/neu/radar.js [hunk 9]
```diff
@@ -0,0 +1,13 @@
...
+        '.': [
+            {
+                title: '学院新闻',
```
Comment (TonyRL) [inline lib/v2/neu/radar.js:6]:
```suggestion
                title: '学院新闻 - 医学与生物信息工程学院',
```
`title` should match the H3 header

File: lib/v2/neu/radar.js [hunk 10]
```diff
@@ -0,0 +1,13 @@
+module.exports = {
+    'www.bmie.neu.edu.cn': {
```
Comment (TonyRL) [inline lib/v2/neu/radar.js:2]:
```suggestion
    'neu.edu.cn': {
```

File: lib/v2/neu/radar.js [hunk 11]
```diff
@@ -0,0 +1,13 @@
...
+        _name: '东北大学医学与生物信息工程学院',
+        '.': [
```
Comment (TonyRL) [inline lib/v2/neu/radar.js:4]:
```suggestion
        'www.bmie': [
```

**Pull request merged**